etc/output_files.yaml

```diff
@@ -197,9 +197,14 @@ gatk_var_calling:
 
 
 project_process:
-  - location: ['']
-    basename: '{project_id}_genotype_gvcfs.vcf'
+    genotype_gvcfs:
+        location: ['']
+        basename: '{project_id}_genotype_gvcfs.vcf.gz'
 
-  - location: ['']
-    basename: '{project_id}.relatedness2'
+    relatedness:
+        location: ['']
+        basename: '{project_id}.relatedness2'
 
+    ped_check:
+        location: ['']
```

Comment:
Currently Output() in projects.py uses location. I'm considering changing this part but for now it is required I think.